pkgs/applications/audio/sound-juicer/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchurl, pkgconfig, gtk3, intltool, itstool, libxml2, brasero, libcanberra_gtk3, gnome3, gst_all_1, libmusicbrainz5, libdiscid, isocodes, makeWrapper }:
```

Comment:
This line could be split, but it's not important, just to let you know when/if you will be updating these packages.